DogWalkerMVC/Controllers/OwnerController.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DogWalkerMVC.Models;
+using DogWalkerMVC.Models.ViewModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Configuration;
+
+namespace DogWalkerMVC.Controllers
+{
+    public class OwnersController : Controller
+    {
+
+        private readonly IConfiguration _config;
+
+        public OwnersController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public SqlConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+        // GET: Owner
+        public IActionResult Index()
+
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT o.[Name] AS OwnerName, n.[Name] AS 'Neighborhood Name', Count(d.[Name]) AS 'Number of Dogs'
+                        FROM Owner o
+                        LEFT JOIN Neighborhood n 
+                        ON n.Id = o.NeighborhoodId
+                        LEFT JOIN Dog d 
+                        ON d.OwnerId = o.Id 
+                        GROUP BY o.[Name], n.[Name]";
```

Comment:
Awesome! Query looks great. I'm thinking We'll probably also need the owner's `Id` too though. It's not immediately obvious from the mockup but if I'm looking at figma correctly, you can click an owner's name and it goes to that owner's detail page. Is that correct @brendalong ? 

@namitamanohar if that _is_ the case, we'll need the Owner's Id to be able to create that `<a>` tag

Reply:
Can this branch be merged? As we have not created the details view for the owners

Reply:
Yeah I'm gonna go ahead and merge it. We'll probably eventually need to add that tho